lighthouse-core/scripts/open-devtools.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+##
+# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+if [[ -z $CHROME_PATH ]]
+then
+  echo 'Must set $CHROME_PATH'
+  exit 1
+fi
+
+export DEVTOOLS_PATH=${DEVTOOLS_PATH:-"$HOME/src/devtools/devtools-frontend"}
```

Comment:
seems oddly specific to someone's dir structure that uses `src` for all projects ;)

can we default to the tmp download of devtools frontend we use for tests so when they fail contributors can run this to open and see what's up? :D

Reply:
ps I use [direnv](https://direnv.net/) to keep personal defaults out of source control, if that helps assuage your concerns :)

Reply:
I think it's the exact path that the standalone devtools checkout guide tells you to use. Do you know a good bash-ism for trying a list of paths in order until one resolves?

Reply:
+1 as an amateur devtools builder/tester, my use case probably 100% of the time will be opening the thing that just failed in `yarn test-devtools`